pytorch_lightning/loggers/base.py

```diff
@@ -296,6 +306,10 @@ def log_hyperparams(self, params: Union[Dict[str, Any], Namespace]) -> None:
         for logger in self._logger_iterable:
             logger.log_hyperparams(params)
 
+    def log_graph(self, model: LightningModule) -> None:
```

Comment:
can we add an optional input argument here, which defaults to the `example_input_array` if None?

Reply:
Done :)